docs/docs/configuration/live.md

```diff
@@ -17,27 +17,37 @@ Live view options can be selected while viewing the live stream. The options are
 
 ### WebRTC extra configuration:
 
-webRTC works by creating a websocket connection on extra ports. One of the following is required for webRTC to work:
-* Frigate is run with `network_mode: host` to support automatic UDP port pass through locally and remotely. See https://github.com/AlexxIT/go2rtc#module-webrtc for more details
-* Frigate is run with `network_mode: bridge` and has:
-    * Router setup to forward port `8555` to port `8555` on the Frigate device.
-    * For local webRTC, you will need to create your own go2rtc config:
-
-```yaml
-log:
-  format: text
-
-webrtc:
-  candidates:
-    - <Frigate host ip address>:8555 # <--- enter Frigate host IP here
-    - stun:8555
-```
-
-and pass that config to Frigate via docker or `frigate-go2rtc.yaml` for addon users:
-
-See https://github.com/AlexxIT/go2rtc#module-webrtc for more details
-
-```yaml
-volumes:
-  - /path/to/your/go2rtc.yaml:/config/frigate-go2rtc.yaml:ro
-```
+WebRTC works through by creating a TCP or UDP connection on port `8555`. However, it requires additional configuration:
+
+* For external access, over internet, setup your router to forward port `8555` to port `8555` on the Frigate device.
```

Comment:
We should mention the port forward should be tcp and udp

Reply:
```suggestion
* For external access, over internet, setup your router to forward port `8555` to port `8555` on the Frigate device, for both TCP and UDP.
```